website/content/getting-started/deployment.mdx

```diff
@@ -0,0 +1,277 @@
+---
+id: getting-started-deployment
+title: Deployment - Getting started
+permalink: /getting-started/deployment
+---
+
+import { ContentNotification } from '@commercetools-frontend/ui-kit';
+
+<ContentNotification type="info">
```

Comment:
Using this component is a test, to see how it looks. If we do decide to use that, we would need to setup theming in order to override some of the styles.